### PR TITLE
Review edits

### PR DESCRIPTION
**Pull request by jwgmeligmeyling**

### CODE REVIEW

File: Documentation/General/productPlanning/planning.tex [hunk 1]
```diff
@@ -145,10 +145,17 @@ \chapter{Definition of Done}
...
+A feature is completed when it is implemented according to its specification and the code is well tested using unit tests.
+Furthermore, the public API (the public methods) of introduced classes should be well-documented using JavaDoc or NgDoc and follow our checkstyle and markup rules.
+Finally, the feature must be reviewed by at least one team member that was not directly involved in creating the code and tests.
+If all these prerequisites are met, the pull request can be merged.\\
 
-A sprint is considered done when the application is in a working, deliverable state. This entails that all features in the codebase are done (as described above). The system also has to be manually tested for the main interaction with the system. Lastly, integration tests are created and ran succesfully.\\
+We will use sprint iterations of one week. All stories assigned for a sprint should be done at the end of a sprint.
+% If a story could not be completed in the sprint, it will be copied to the next sprint and this will be discussed in the sprint reflection in order to prevent planning failures in the future.
```
Comment (eanker) [inline Documentation/General/productPlanning/planning.tex]: I like this,I say let's uncomment this.

**Comment by eanker** [comment]
Good branch, just one comment.
I would say merge, if we have discust this.

File: Documentation/General/productPlanning/planning.tex [hunk 2]
```diff
@@ -145,10 +145,17 @@ \chapter{Definition of Done}
...
+The system also has to be manually tested for the main interaction with the system. Lastly, integration tests are created and ran succesfully.\\
 
-The final product is considered done if it passes a few criteria. First of all, the system must be unit and integration tested, and it must be manually inspected for all common user interactions. All must-have requirements must be implemented. If a should-have requirement is not implemented, this has to be agreed upon by all team members with a valid reason, and the product should still be usable without the feature. Code is consistent, validated by the SIG test and documented.
+The final product is done when all requirements are fulfilled and the implementation matches the specification. The implementation should be well tested using unit tests and preferably integration / end-to-end tests as well.
```
Comment (TimvdLippe) [inline Documentation/General/productPlanning/planning.tex:157]: 'The final product can be released' a product is never done/finished.

File: Documentation/General/productPlanning/requirements.tex [hunk 3]
```diff
@@ -30,14 +30,12 @@ \subsubsection{Must haves}
 		\item Bored
 		\item Sad
 		\end{itemize}
-	  These moods are defined by the Arousal/Valence model\footnote{Y. Chang and H.H. Chen. Music Emotion Recognition. Boca raton, State:
```
Comment (TimvdLippe) [inline Documentation/General/productPlanning/requirements.tex:33]: What happened with this citation, why was it removed?

File: Documentation/General/productPlanning/requirements.tex [hunk 4]
```diff
@@ -46,32 +44,27 @@ \subsubsection{Must haves}
 \end{itemize}
 \subsubsection{Should haves}
 \begin{itemize}
-\item The web interface shall start playing the current song on the current timestamp when a user joins the room.
+\item Users in the same room listen to the same music at the same time point.
```
Comment (TimvdLippe) [inline Documentation/General/productPlanning/requirements.tex]: 'the same song' sounds better to me.

Comment (TimvdLippe) [inline Documentation/General/productPlanning/requirements.tex]: 'at the same point in time'

File: Documentation/General/productPlanning/requirements.tex [hunk 5]
```diff
@@ -46,32 +44,27 @@ \subsubsection{Must haves}
...
 	\begin{itemize}
 	\item The song received $\frac{\$number\_of\_users\_in\_room}{3}$ votes to skip.
 	\item The song received at least 10 votes, and the ratio upvote:downvote is 1:2 or less.
-	\item A moderator skips the song (to prevent escalations)
 	\end{itemize}
-	
-\item The system shall periodically suggest the user to hop to another room to encourage more interaction and we can direct the user to a more preferred mood.
 
-\item The multi-layer perceptron shall be trained with metadata provided by EEMCS
+\item The multi-layer perceptron will be enabled by generating initial values from the metadata in our dataset.
```
Comment (TimvdLippe) [inline Documentation/General/productPlanning/requirements.tex]: 'will be initialized' enabling can also happen after it was initialized.

File: Documentation/General/productPlanning/planning.tex (hunk 2 above)
Comment (jwgmeligmeyling) [inline Documentation/General/productPlanning/planning.tex:157]: Discussed in the other pr

**Pull request merged**